src/Compilers/CSharp/Test/Emit/CSharpCompilerEmitTest.csproj

```diff
@@ -55,9 +56,6 @@
     <Reference Include="..\..\..\..\..\packages\System.Reflection.Metadata.$(SystemReflectionMetadataVersion)\lib\portable-net45+win8\System.Reflection.Metadata.dll" />
     <Reference Include="..\..\..\..\..\packages\System.Collections.Immutable.$(SystemCollectionsImmutableVersion)\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll" />
     <Reference Include="..\..\..\..\..\packages\Microsoft.DiaSymReader.1.0.5\lib\net45\Microsoft.DiaSymReader.dll" />
-    <Reference Include="xunit">
-      <HintPath>..\..\..\..\..\packages\xunit.1.9.2\lib\net20\xunit.dll</HintPath>
-    </Reference>
```

Comment:
It seems, the current recommendation (by @jaredpar) is:
* Modify VSL.Settings.targets to include new flags for XunitAssemblyVersion and XunitVersion (as well as any specifies for each other xunit dependency; so XunitAttributesAssemblyVersion and XunitAttributesVersion, XunitCoreAssemblyVersion and XunitCoreVersion, etc..)
* Modify each UnitTest assembly project to include a new flag (such as <UnitTestAssembly>true<UnitTestAssembly>)
* Modify each UnitTest assembly and remove the existing reference to Roslyn.Toolsets.Xunit.targets
* Modify each UnitTest assembly to directly reference the xUnit dependencies and use the Xunit*AssemblyVersion/Xunit*Version flags for versioning
* Modify VSL.Imports.targets to import xunit.core.props and Roslyn.Toolsets.xunit.targets if the UnitTestAssembly flag is defined as true.

Is that about right?